analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.NetCore.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
```

Comment:
I already created some NetCore specific tests in `NullPointerDereference.CSharp8.cs` (e.g. tests in `ThrowHelper` and `NotNullWhenTests`). The reasoning was, that NRT was introduced along with C#8 and with it the annotation attributes. So the annotation attributes are tied to .Net Core 5 and .NetStandard 2.1 and C#8 is the oldest official supported language version for these runtimes.
I think it is good to have `NullPointerDereference.NetFx.cs` but `NullPointerDereference.NetCore.cs` looks like a duplicate to `NullPointerDereference.CSharp8.cs` to me.

Reply:
The problem is that `CSharp8.cs` is missused to test `NetCore`-only stuff.

C# 8 is language version and needs to be tested under NetFx too.
I've moved the test cases from there to `.NetCore.cs`, independent on language version. And made the C# 8 UT run under both TFMs